LL/corco_bascind.trig

```diff
@@ -85,26 +85,24 @@
     a foaf:Person;
     irishRel:genName "Decci";
     irishRel:nomName "Decc";
+    foaf:knows <http://www.wikidata.org/entity/Q165479> ;
+    rdfs:comment "quem Patricius baptizauit" ;            
     rel:childOf <#Imchada>.
 
 <#Imchada>
     a foaf:Person;
     irishRel:genName "Imchada";
     irishRel:nomName "Imchad";
     irishRel:numChild 5;
-    rel:childOf <#Cuirp-fdb5bfa8>.
+    rel:childOf <#Cuirp-fdb5bfa8>, <#Imchad-16716b50>.
```

Comment:
Is Imchad really child of himself? sameAs?

Reply:
Right... nice catch.

Reply:
Yeah.... good catch.